### PR TITLE
feat(Rebranding / Elevations): add elevation tokens

### PR DESCRIPTION
**Pull request by saurabhdaware**
add elevation tokens for web and native

For web, we use same values as figma
For native, I and Abhishek got on call to create shadows as close to figma as possible but it cannot be same because native (especially android) has very limited things we can change in shadow 

Web - iOS - Android

image

### CODE REVIEW

File: packages/blade/src/tokens/global/elevation/elevation.web.ts [hunk 1]
```diff
@@ -4,15 +4,15 @@ export const elevation: ElevationWithColorModes = {
...
-    lowRaised: `0 4px 8px -2px hsla(217, 56%, 17%, 0.1), 0 2px 4px -2px hsla(217, 56%, 17%, 0.06)`,
-    midRaised: `0 12px 16px -4px hsla(217, 56%, 17%, 0.08), 0 4px 6px -2px hsla(217, 56%, 17%, 0.03)`,
-    highRaised: `0 24px 48px -12px hsla(217, 56%, 17%, 0.18)`,
+    lowRaised: `0px 2px 16px 0px hsla(217, 56%, 17%, 0.10)`,
+    midRaised: `0px 8px 24px 0px hsla(217, 56%, 17%, 0.12)`,
+    highRaised: `0px 16px 48px -12px hsla(217, 56%, 17%, 0.18)`,
   },
   onDark: {
     none: 'none',
     // @TODO: replace the values with correct dark mode values
```
[the remark this answers is not in the document]
Comment (saurabhdaware) [reply packages/blade/src/tokens/global/elevation/elevation.web.ts:13]: Needs to be changed in design. RK was discussing it that day but yes not ready from design yet.

**Comment by saurabhdaware** [comment]
> LGTM, let's test things out on RN

done

**Comment by kamleshchandnani** [comment]
@saurabhdaware can you post screenshots of web+iOS+android of before and after in the description?

**Comment by kamleshchandnani** [comment]
we also need to update [these](https://61c19ee8d3d282003ac1d81c-ffisbzkbbb.chromatic.com/?path=/docs/tokens-elevation--page&globals=showInternalComponents:true) right?

**Comment by kamleshchandnani** [comment]
the mid and high raised in iOS looks too much elevation and black shadows spread might just make the UI feel cluttered. Probably we shall revisit.

**Comment by saurabhdaware** [comment]
> the mid and high raised in iOS looks too much elevation and black shadows spread might just make the UI feel cluttered. Probably we shall revisit.

Yup agree. The problem was Android and iOS are using the same opacity value and for some reason they look entirely different 🤦🏼 . Might have to add a check and use different values for android and ios. Lemme check.

**Comment by saurabhdaware** [comment]
@kamleshchandnani 

- Added the screenshot
- No need of updating the documentation since it loops through the tokens already
- Improved the shadows a bit. They are a bit closer to each other on different platforms now. Definitely not same but not sure if that is possible.

**Review by kamleshchandnani — APPROVED** [review]

**Pull request merged**